sspl_test/run_qa_test.sh

```diff
@@ -228,6 +243,15 @@ then
             conf $global_config set "storage_enclosure>$encl_id>controller>primary>ip=$MOCK_SERVER_IP"
         fi
     fi
+    if [ "${CVG_INFO}" == "null" ]
+        then
+        CVG_NAME="null"
+        `conf $global_config set "server_node>$machine_id>storage>cvg[0]>name=${MOCK_CVG_NAME}"`
+    else
+        CVG_NAME=`conf $global_config get "server_node>$machine_id>storage>cvg[0]>name"`
+        CVG_NAME=$(echo $CVG_NAME | tr -d "["\" | tr -d "\"]")
+        `conf $global_config set "server_node>$machine_id>storage>cvg[0]>name=${MOCK_CVG_NAME}"`
```

Comment:
This line is common in both if and else block, can we define outside of if-else.

Reply:
Done